src/main/java/frc/robot/subsystems/swerve/SwerveType.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.swerve;
+
+public enum SwerveType {
+
+	SWERVE("");
```

Comment:
why do you need the ""?

Reply:
for now we have one swerve so the name should be swerve